Mathlib/Topology/MetricSpace/MetricSeparated.lean

```diff
@@ -20,51 +20,54 @@ open EMetric Set
 
 noncomputable section
 
+namespace Metric
```

Comment:
Wouldn't `EMetric` be more natural?

Reply:
Since there's no analogous metric-specific concept, the convention around this part of the library seems to say that it should be in the `Metric` namespace.